GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignInButton.h

```diff
@@ -19,26 +19,11 @@
 
 #import <UIKit/UIKit.h>
 
-NS_ASSUME_NONNULL_BEGIN
-
-/// The layout styles supported by the `GIDSignInButton`.
-///
-/// The minimum size of the button depends on the language used for text.
-/// The following dimensions (in points) fit for all languages:
-/// - kGIDSignInButtonStyleStandard: 230 x 48
-/// - kGIDSignInButtonStyleWide:     312 x 48
-/// - kGIDSignInButtonStyleIconOnly: 48 x 48 (no text, fixed size)
-typedef NS_ENUM(NSInteger, GIDSignInButtonStyle) {
-  kGIDSignInButtonStyleStandard = 0,
-  kGIDSignInButtonStyleWide = 1,
-  kGIDSignInButtonStyleIconOnly = 2
-};
+// Forward declarations of enums
```

Comment:
If we don't define the enum values here, is this not a breaking change?

Reply:
I've imported `GIDSignInButtonStyling.h` into `GIDSignInButton.h`  to circumvent this. Per offline chat, I agree that this isn't desirable as we're leaking implementation details into our API, but I cannot see a way of creating a Swift target that depends upon this information without doing something like that. We can treat this as a temporary solution and create an issue to track fixing this.

Reply:
Discussed offline and agreed to pursue a refactor with the goal of not having any public API surface impact.